xbmc/GUIInfoManager.cpp

```diff
@@ -188,6 +188,18 @@ typedef struct
   int  val;
 } infomap;
 
+const infomap string_bools[] =   {{ "isempty",          STRING_IS_EMPTY },
+                                  { "equals",           STRING_EQUALS },
+                                  { "startswith",       STRING_STARTS_WITH },
+                                  { "endswith",         STRING_ENDS_WITH },
+                                  { "contains",         STRING_CONTAINS }};
+
+const infomap integer_bools[] =  {{ "equals",           INTEGER_EQUALS },
+                                  { "greater",          INTEGER_GREATER_THAN },
+                                  { "greaterorequals",  INTEGER_GREATER_OR_EQUALS },
+                                  { "less",             INTEGER_LESS_THAN },
+                                  { "lessorequals",     INTEGER_LESS_OR_EQUALS }};
```

Comment:
They are very verbose. Perhaps use the de facto standard  eq, gt, ge, lt, le notation?

Also, why not 'not equal`?